src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -225,6 +229,16 @@ private static SimpleNameSyntax GetLeftMostSimpleName(QualifiedNameSyntax qn)
             return semanticModel.GetTypeInfo(node).Type;
         }
 
+        protected override ITypeSymbol GetAwaitInfo(
+            SemanticModel semanticModel, SyntaxNode node, CancellationToken cancellationToken)
+        {
+            var @await = node.AncestorsAndSelf().OfType<AwaitExpressionSyntax>().First();
```

Comment:
❓ Any reason to not use `node.FirstAncestorOrSelf<AwaitExpressionSyntax>()`?

💡 `var awaitExpression`

Reply:
Admittedly, I just took that line from the `GetQueryClauseInfo()` implementation below mine. Apologies, I should have realized they did the same thing and used your suggestion to begin with